Resources/config/console.xml

```diff
@@ -9,6 +9,17 @@
             <argument type="service" id="lexik_jwt_authentication.key_loader" />
             <tag name="console.command" command="lexik:jwt:check-config" />
         </service>
+
+        <service id="lexik_jwt_authentication.user_provider_locator">
```

Comment:
should be explicitly `public="false"` to not lose DI performance in older symfony versions?

Reply:
It can yep. Changed 👍